### DISCUSSION TITLE
Skywalking 8.9.1 version - UI Observations

### DISCUSSION THREAD

**Discussion by lstephe** [apache/skywalking#9544] [General]
The below observations with Skywalking 8.9.1 UI:

1.   Exception while fetching data (/getEndpoints) : com.linecorp.armeria.client.ResponseTimeoutException.   What causes this?  Recommendation on how to resolve?

2.  Trace page, the service dropdown not loading.  There is no error.  Have to refresh the page.  What causes this?  Recommendation on how to resolve?

3.  Are the above two UI observations addressed in 9.X version?

**Comment by wu-sheng** [comment]
(1) looks like a database level error. I can't see the details.
(2) Never faced this.

SkyWalking community has moved on 9.x, which UI is totally different.
